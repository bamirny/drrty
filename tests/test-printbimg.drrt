def int main() {
	createElement("img","imgClass","https://as1.ftcdn.net/jpg/00/33/17/90/220_F_33179059_DAZeqcO1edV3WBODcrL9RGQ39todSjBV.jpg");

	return 0;
	
}