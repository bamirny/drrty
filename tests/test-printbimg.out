<img class="imgClass" src="https://as1.ftcdn.net/jpg/00/33/17/90/220_F_33179059_DAZeqcO1edV3WBODcrL9RGQ39todSjBV.jpg">
